docs/source/api/samplers.rst

```diff
@@ -30,6 +30,8 @@ HMC family
 
    NUTS
    HamiltonianMC
+   sampling_jax.sample_blackjax_nuts
```

Comment:
I think they should be in the first block (or in a different subsection), IIUC, both are nuts only even if blackjax has other samplers, but they are not step methods that can be used as inputs to pm.sample